diff --git a/node_modules/expo-json-utils/android/build.gradle b/node_modules/expo-json-utils/android/build.gradle
index daa4fa4..0f6050c 100644
--- a/node_modules/expo-json-utils/android/build.gradle
+++ b/node_modules/expo-json-utils/android/build.gradle
@@ -7,14 +7,14 @@ group = 'host.exp.exponent'
 version = '0.15.0'
 
 expoModule {
-  canBePublished false
+  canBePublished = false
 }
 
 android {
-  namespace "expo.modules.jsonutils"
+  namespace = "expo.modules.jsonutils"
   defaultConfig {
-    versionCode 31
-    versionName '0.15.0'
+    versionCode = 31
+    versionName = '0.15.0'
     testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
   }
   testOptions {
