diff --git a/node_modules/expo-manifests/android/build.gradle b/node_modules/expo-manifests/android/build.gradle
index 708c0b7..dc49b73 100644
--- a/node_modules/expo-manifests/android/build.gradle
+++ b/node_modules/expo-manifests/android/build.gradle
@@ -7,14 +7,14 @@ group = 'host.exp.exponent'
 version = '1.0.10'
 
 expoModule {
-  canBePublished false
+  canBePublished = false
 }
 
 android {
-  namespace "expo.modules.manifests"
+  namespace = "expo.modules.manifests"
   defaultConfig {
-    versionCode 31
-    versionName '1.0.10'
+    versionCode = 31
+    versionName = '1.0.10'
     testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
   }
   testOptions {
