diff --git a/node_modules/expo-dev-client/android/build.gradle b/node_modules/expo-dev-client/android/build.gradle
index a2517f5..8143697 100644
--- a/node_modules/expo-dev-client/android/build.gradle
+++ b/node_modules/expo-dev-client/android/build.gradle
@@ -4,17 +4,17 @@ plugins {
 }
 
 expoModule {
-  canBePublished false
+  canBePublished = false
 }
 
 group = "host.exp.exponent"
 version = "6.0.20"
 
 android {
-  namespace "expo.modules.devclient"
+  namespace = "expo.modules.devclient"
   defaultConfig {
-    versionCode 1
-    versionName "6.0.20"
+    versionCode = 1
+    versionName = "6.0.20"
     testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
   }
 
@@ -25,8 +25,8 @@ android {
   }
 
   buildFeatures {
-    buildConfig true
-    viewBinding true
+    buildConfig = true
+    viewBinding = true
   }
 
   packaging {
