diff --git a/node_modules/expo-dev-launcher/android/build.gradle b/node_modules/expo-dev-launcher/android/build.gradle
index 33f4e9c..eadee22 100644
--- a/node_modules/expo-dev-launcher/android/build.gradle
+++ b/node_modules/expo-dev-launcher/android/build.gradle
@@ -16,17 +16,17 @@ apply plugin: 'org.jetbrains.kotlin.plugin.serialization'
 apply plugin: 'com.apollographql.apollo'
 
 expoModule {
-  canBePublished false
+  canBePublished = false
 }
 
 group = "host.exp.exponent"
 version = "6.0.20"
 
 android {
-  namespace "expo.modules.devlauncher"
+  namespace = "expo.modules.devlauncher"
   defaultConfig {
-    versionCode 9
-    versionName "6.0.20"
+    versionCode = 9
+    versionName = "6.0.20"
   }
 
   buildTypes {
@@ -47,8 +47,8 @@ android {
   }
 
   buildFeatures {
-    buildConfig true
-    viewBinding true
+    buildConfig = true
+    viewBinding = true
   }
 }
 
@@ -66,11 +66,11 @@ repositories {
   mavenLocal()
   maven {
     // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-    url "$rootDir/../node_modules/react-native/android"
+    url = "$rootDir/../node_modules/react-native/android"
   }
   maven {
     // Android JSC is installed from npm
-    url "$rootDir/../node_modules/jsc-android/dist"
+    url = "$rootDir/../node_modules/jsc-android/dist"
   }
   google()
 }
