diff --git a/node_modules/react-native-quick-base64/android/build.gradle b/node_modules/react-native-quick-base64/android/build.gradle
index 6a104e7..1537875 100644
--- a/node_modules/react-native-quick-base64/android/build.gradle
+++ b/node_modules/react-native-quick-base64/android/build.gradle
@@ -38,15 +38,15 @@ def resolveBuildType() {
 }
 
 android {
-  namespace 'com.reactnativequickbase64'
+  namespace = 'com.reactnativequickbase64'
 
   compileSdkVersion getExtOrIntegerDefault("compileSdkVersion")
 
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault("minSdkVersion")
     targetSdkVersion getExtOrIntegerDefault("targetSdkVersion")
-    versionCode 1
-    versionName "1.0"
+    versionCode = 1
+    versionName = "1.0"
     externalNativeBuild {
       cmake {
         cppFlags "-O2 -frtti -fexceptions -Wall -Wno-unused-variable -fstack-protector-all"
@@ -57,8 +57,8 @@ android {
   }
 
   buildFeatures {
-    buildConfig true
-    prefab true
+    buildConfig = true
+    prefab = true
   }
 
   buildTypes {
