diff --git a/node_modules/react-native-safe-area-context/android/build.gradle b/node_modules/react-native-safe-area-context/android/build.gradle
index f324b9a..7c558fd 100644
--- a/node_modules/react-native-safe-area-context/android/build.gradle
+++ b/node_modules/react-native-safe-area-context/android/build.gradle
@@ -41,9 +41,9 @@ if (isNewArchitectureEnabled()) {
 android {
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
     if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
-        namespace "com.th3rdwave.safeareacontext"
+        namespace = "com.th3rdwave.safeareacontext"
         buildFeatures {
-            buildConfig true
+            buildConfig = true
         }
     }
 
@@ -55,14 +55,14 @@ android {
         ndkPath rootProject.ext.ndkPath
     }
     if (rootProject.hasProperty("ndkVersion")) {
-        ndkVersion rootProject.ext.ndkVersion
+        ndkVersion = rootProject.ext.ndkVersion
     }
 
     defaultConfig {
         minSdkVersion getExtOrDefault('minSdkVersion', 16)
         targetSdkVersion getExtOrDefault('targetSdkVersion', 28)
-        versionCode 1
-        versionName "1.0"
+        versionCode = 1
+        versionName = "1.0"
         buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
 
         ndk {
@@ -71,7 +71,7 @@ android {
     }
 
     lintOptions{
-        abortOnError false
+        abortOnError = false
     }
 
     packagingOptions {
@@ -111,7 +111,7 @@ repositories {
     google()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        url "$rootDir/../node_modules/react-native/android"
+        url = "$rootDir/../node_modules/react-native/android"
     }
     mavenCentral()
 }
